web-app/HeimScripts/heatmap/summary.R

```diff
@@ -147,6 +198,9 @@ produce_boxplot <- function(measurement_tables, phase)
     measurement_vectors[[i]] <- unlist(measurement_vectors[[i]]) 
```

Comment:
Yes, in loaded variables the label of each dataframe is of format n<node_identifier>_s<subset_identifier>, where the identifiers are numerical